src/components/AppBar/index.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import CssBaseline from '@mui/material/CssBaseline';
+import Button from '@mui/material/Button';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { LinkItem } from '../../types';
+import styles from './styles.module.scss';
+import cs2img from '../../assets/cs2img.webp';
+import valorant from '../../assets/valorant.jpg';
+import logo from '../../assets/logo.png';
+
+type AppBarProps = React.PropsWithChildren<{
+  links?: LinkItem[];
+}>;
+
+const AppBar: React.FC<AppBarProps> = ({
+  children,
+  links = [
```

Comment:
it's should not be in AppBar component props since you're not using it anywhere, just create constant before return method here